src/components/cylc/tree/Tree.vue

```diff
@@ -160,7 +160,7 @@ import TreeItem from '@/components/cylc/tree/TreeItem'
 import Vue from 'vue'
 import TaskState from '@/model/TaskState.model'
 import Task from '@/components/cylc/Task'
-import clonedeep from 'lodash.clonedeep'
+import cloneDeep from 'lodash/cloneDeep'
```

Comment:
The first one was using the [`lodash.clonedeep`](https://www.npmjs.com/package/lodash.clonedeep) single dependency, which was a transitive dependency.

Our build failed as Yarn is more strict about the transitive dependencies not declared in our `package.json` (we had similar issue in Python & setuptools I think? Maybe a trend in build tools? anyhoo).

Instead of simply adding the new dependency, the change here is using importing the `cloneDeep` module from our already imported `lodash` dependency. Tree shaking doesn't seem to work too well for Lodash at the moment with our Vue-CLI/Babel/Webpack build chain, but it does with RollUp and with with Vite, which we will probably have a chance to look into when moving to Vue 3.